format_test.go

```diff
@@ -67,7 +67,7 @@ func Test_format(t *testing.T) {
 			name: "case 4: as case 2 but grep for level==warning && function==GetCurrentState",
 			text: `{"caller":"github.com/giantswarm/cluster-operator/vendor/github.com/giantswarm/operatorkit/controller/resource/retryresource/crud_resource_ops_wrapper.go:71","event":"update","function":"GetCurrentState","level":"warning","message":"retrying due to error","object":"/apis/core.giantswarm.io/v1alpha1/namespaces/default/kvmclusterconfigs/h0cg9-kvm-cluster-config","resource":"chartv2","stack":"[{/go/src/github.com/giantswarm/cluster-operator/vendor/github.com/giantswarm/operatorkit/controller/resource/retryresource/crud_resource_ops_wrapper.go:64: } {/go/src/github.com/giantswarm/cluster-operator/pkg/v2/resource/chart/current.go:37: } {/go/src/github.com/giantswarm/cluster-operator/pkg/v2/resource/chart/resource.go:126: } {/go/src/github.com/giantswarm/cluster-operator/vendor/github.com/giantswarm/helmclient/helmclient.go:236: rpc error: code = Unimplemented desc = unknown service grpc.health.v1.Health} {Tiller installation failed}]","time":"2018-06-01T06:26:15.261267+00:00","resource":"chartv2"}`,
 			grep: "level=warning,function=GetCurrentState",
-			expectedOut: `W 18-06-01T06:26:15.261267+0 /apis/core.giantswarm.io/v1alpha1/namespaces/default/kvmclusterconfigs/h0cg9-kvm-cluster-config chartv2.GetCurrentState retrying due to error | operatorkit/controller/resource/retryresource/crud_resource_ops_wrapper.go:71 | event=update
+			expectedOut: `W 06/01 06:26:15 /apis/core.giantswarm.io/v1alpha1/namespaces/default/kvmclusterconfigs/h0cg9-kvm-cluster-config chartv2.GetCurrentState retrying due to error | operatorkit/controller/resource/retryresource/crud_resource_ops_wrapper.go:71 | event=update
```

Comment:
I feel like we don't need millis, `+0` - I'm not even sure that that is, probably UTC indication. This saves couple of characters and makes date clearer (at least for me).

Reply:
Yeah, I think `+0` is for time zone.